packages/app-store/make/pages/setup/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { Trans } from "next-i18next";
+import Link from "next/link";
+import { useState } from "react";
+import { Toaster } from "react-hot-toast";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Button, Tooltip, showToast } from "@calcom/ui";
+import { Clipboard } from "@calcom/ui/components/icon";
+
+const MAKE = "make";
+const appInviteLink = process.env.MAKE_INVITE_LINK; //TODO Add MAKE_INVITE_LINK to env vars
```

Comment:
We would need to get it from `appKeys` like [this](https://github.com/aar2dee2/cal.com/blob/6bfabe41e5073ff3a9e38d577ef6d72be2d6eca8/packages/app-store/jitsivideo/lib/VideoApiAdapter.ts#L16) as we are not setting in env now.

Reply:
Sorted this one.

Reply:
seems like you fixed this?

Reply:
Yeah.